flutter/realm_flutter/ios/realm.podspec

```diff
@@ -37,4 +37,9 @@ Pod::Spec.new do |s|
                                   ],
                                   'FRAMEWORK_SEARCH_PATHS' => '"$(PODS_TARGET_SRCROOT)/**"'
                                 }
+  s.script_phase              = { :name => 'Report Metrics', 
+                                  # Cannot use $FLUTTER_APPLICATION_PATH (it is not exported), so use $PROJECT_DIR/../.. instead
```

Comment:
```suggestion
                                  
```

Reply:
just a suggestion that maybe this comment is not needed. I certainly do not expect Flutter_app_path here. Feel free leave it if you think it is needed.